web/src/main/webapp/xslt/services/inspire-atom/opensearch.xsl

```diff
@@ -53,21 +53,29 @@
             </xsl:attribute>
           </Url>
 
-
           <!--Generic URL template for browser integration-->
           <Url type="application/xml" rel="results">
             <xsl:attribute name="template">
               <xsl:value-of
                 select="concat(//server/protocol,'://',//server/host,':',//server/port,/root/gui/url,'/opensearch/', /root/gui/language, '/', /root/response/fileId,'/search?any={filter}')"/>
```

Comment:
Please check if should be checked the protocol to use `//server/port`or `//server/securePort`